BangazonAPI/Models/ComputerEmployee.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class ComputerEmployee
+    {
+        public int id { get; set; }
+        public DateTime assignDate { get; set; }
```

Comment:
You don't need this model-- we never need to see any info from this join table (although we will need to reference it in our SQL queries)